dataprofiler/labelers/column_name_model.py

```diff
@@ -220,8 +229,8 @@ def predict(
         if show_confidences:
             raise NotImplementedError(
                 """`show_confidences` parameter is disabled
-                for Proof of Concept implementation. Confidence
-                values are enabled by default."""
+                for MVP implementation. Note: Confidence
+                values are returned by default."""
```

Comment:
My thought is we look at this in a re-work of the pipeline. Will require some thought how the data is passed into the labeler flow and re formatting of the `true_positive_dict` and `false_positive_dict` parameter formatting